uw_sws/models.py

```diff
@@ -486,7 +486,9 @@ class Section(models.Model):
     sln = models.PositiveIntegerField()
     summer_term = models.CharField(max_length=12, null=True)
     delete_flag = models.CharField(max_length=20)
-    is_withdrawn = models.NullBooleanField()
+    is_active = models.NullBooleanField(default=False)
```

Comment:
Instead of adding three boolean fields, how about just adding three functions?

Reply:
I added two booleans, is_withdrawn was already there and I didnt want to break existing apps.

Reply:
Actually, is_withdrawn was already there.  How about if I change these to functions, and we can bump the next release to 1.7, since there will be a breaking change with is_withdrawn?